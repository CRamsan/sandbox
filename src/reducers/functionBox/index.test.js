import reducer from '.'

const initialState = {
  position: { x: 0.65, y: 1.45, z: -1.03 },
}

describe('functionBox', () => {
  it('should return the initial state', () => {
    const next = reducer(initialState, {})
    expect(next).toEqual(initialState)
  })

  it('FUNCTION_BOX_SET_POSITION', () => {
    const next = reducer(initialState, { type: 'FUNCTION_BOX_SET_POSITION', position: { x: 0, y: 0, z: 0 } })
    expect(next).toMatchSnapshot()
  })
})
