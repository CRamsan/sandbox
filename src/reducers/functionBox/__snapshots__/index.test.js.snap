// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`functionBox FUNCTION_BOX_SET_POSITION 1`] = `
Object {
  "position": Object {
    "x": 0,
    "y": 0,
    "z": 0,
  },
}
`;
